index.html

```diff
@@ -66,24 +94,45 @@ <h3>Multi-Post Stories</h3>
           <li>boostrap</li>
           <li>ruby</li>
         </ul>
-
+        <div class="button_works">
         <button type="submit">See Project</button>
```

Comment:
- [ ] Please the `See Project` button takes you to the GitHub repository. Kindly link that.